### ISSUE TITLE
Running "./darknet": Permission denied / command not found

### ISSUE THREAD

**Issue by bgaiarin** [pjreddie/darknet#2286]
I have compiled successfully (_no error messages_) on my Google VM running Linux (_Linux 4.9.0-13-amd64 Debian 4.9.228-1 x86_64 GNU/Linux_). 

When I run `./darknet`, I receive the following error:
`-bash: ./darknet: Permission denied`

When I run `sudo ./darknet`, I receive the following error: 
`sudo: ./darknet: command not found`

Any guidance would be appreciated. I've tried changing permissions using chmod (using lines like `chmod +x darknet`), but my efforts there haven't made an impact. Running low on ideas. Thanks!

**Comment by marxenegls** [comment]
did you compile as user in user's own home dir ?
could it be related to the gcloud user/group,.. but typically it would still inherit ownership from $user dir but if it in /etc/sudoers ??
- what if you logged in with a diff user ?
did you create one ??
'sudo' is the command that isn't recognized , right ?
if it compiled than it has to be related to uid/gids , or gcloud has a diff build process and utilities > docs 
what do you think ??
(typically that happens on my end on pi's that experienced a segfault the wrong format not meant for arm (happened a few times)  - requires a reboot and dick check for good measure cmdline.txt option  ..force check raspbian )
gCloud is amd64 - so i don't see how that would be your case : \ 

docs gcloud build : 'https://cloud.google.com/sdk/gcloud/reference/builds '

**Comment by marxenegls** [comment]
$ ./darknet detect cfg/yolov3.cfg /mnt/extdata/TF/yolov3.weights data/dog.jpg
layer     filters    size              input                output
    0 conv     32  3 x 3 / 1   608 x 608 x   3   ->   608 x 608 x  32  0.639 BFLOPs
    1 conv     64  3 x 3 / 2   608 x 608 x  32   ->   304 x 304 x  64  3.407 BFLOPs
    2 conv     32  1 x 1 / 1   304 x 304 x  64   ->   304 x 304 x  32  0.379 BFLOPs
    3 conv     64  3 x 3 / 1   304 x 304 x  32   ->   304 x 304 x  64  3.407 BFLOPs
    4 res    1                 304 x 304 x  64   ->   304 x 304 x  64
    5 conv    128  3 x 3 / 2   304 x 304 x  64   ->   152 x 152 x 128  3.407 BFLOPs
    6 conv     64  1 x 1 / 1   152 x 152 x 128   ->   152 x 152 x  64  0.379 BFLOPs
    7 conv    128  3 x 3 / 1   152 x 152 x  64   ->   152 x 152 x 128  3.407 BFLOPs
    8 res    5                 152 x 152 x 128   ->   152 x 152 x 128
    9 conv     64  1 x 1 / 1   152 x 152 x 128   ->   152 x 152 x  64  0.379 BFLOPs
   10 conv    128  3 x 3 / 1   152 x 152 x  64   ->   152 x 152 x 128  3.407 BFLOPs
   11 res    8                 152 x 152 x 128   ->   152 x 152 x 128
   12 conv    256  3 x 3 / 2   152 x 152 x 128   ->    76 x  76 x 256  3.407 BFLOPs
   13 conv    128  1 x 1 / 1    76 x  76 x 256   ->    76 x  76 x 128  0.379 BFLOPs
   14 conv    256  3 x 3 / 1    76 x  76 x 128   ->    76 x  76 x 256  3.407 BFLOPs
   15 res   12                  76 x  76 x 256   ->    76 x  76 x 256
   16 conv    128  1 x 1 / 1    76 x  76 x 256   ->    76 x  76 x 128  0.379 BFLOPs
   17 conv    256  3 x 3 / 1    76 x  76 x 128   ->    76 x  76 x 256  3.407 BFLOPs
   18 res   15                  76 x  76 x 256   ->    76 x  76 x 256
   19 conv    128  1 x 1 / 1    76 x  76 x 256   ->    76 x  76 x 128  0.379 BFLOPs
   20 conv    256  3 x 3 / 1    76 x  76 x 128   ->    76 x  76 x 256  3.407 BFLOPs
   21 res   18                  76 x  76 x 256   ->    76 x  76 x 256
   22 conv    128  1 x 1 / 1    76 x  76 x 256   ->    76 x  76 x 128  0.379 BFLOPs
   23 conv    256  3 x 3 / 1    76 x  76 x 128   ->    76 x  76 x 256  3.407 BFLOPs
   24 res   21                  76 x  76 x 256   ->    76 x  76 x 256
   25 conv    128  1 x 1 / 1    76 x  76 x 256   ->    76 x  76 x 128  0.379 BFLOPs
   26 conv    256  3 x 3 / 1    76 x  76 x 128   ->    76 x  76 x 256  3.407 BFLOPs
   27 res   24                  76 x  76 x 256   ->    76 x  76 x 256
   28 conv    128  1 x 1 / 1    76 x  76 x 256   ->    76 x  76 x 128  0.379 BFLOPs
   29 conv    256  3 x 3 / 1    76 x  76 x 128   ->    76 x  76 x 256  3.407 BFLOPs
   30 res   27                  76 x  76 x 256   ->    76 x  76 x 256
   31 conv    128  1 x 1 / 1    76 x  76 x 256   ->    76 x  76 x 128  0.379 BFLOPs
   32 conv    256  3 x 3 / 1    76 x  76 x 128   ->    76 x  76 x 256  3.407 BFLOPs
   33 res   30                  76 x  76 x 256   ->    76 x  76 x 256
   34 conv    128  1 x 1 / 1    76 x  76 x 256   ->    76 x  76 x 128  0.379 BFLOPs
   35 conv    256  3 x 3 / 1    76 x  76 x 128   ->    76 x  76 x 256  3.407 BFLOPs
   36 res   33                  76 x  76 x 256   ->    76 x  76 x 256
   37 conv    512  3 x 3 / 2    76 x  76 x 256   ->    38 x  38 x 512  3.407 BFLOPs
   38 conv    256  1 x 1 / 1    38 x  38 x 512   ->    38 x  38 x 256  0.379 BFLOPs
    38 conv    256  1 x 1 / 1    38 x  38 x 512   ->    38 x  38 x 256  0.379 BFLOPs
   39 Segmentation fault !!
spoke too soon 
i got this running without sudo but got your output with sudo 
i guess we've both been had : (

**Comment by bgaiarin** [comment]
Thanks @marxenegls ! Great questions/suggestions. I didn't end up figuring out exactly what was going on with the permissions issue, but this does seem to be a case where Google Fuse (which is used to mount a Google Cloud Storage project in the VM instance) has weird permissions issues. 

I ended up ditching this approach altogether and compiling + running on a VM without GCS mounted, which worked just fine. Line I used is below: 
`./darknet detector demo cfg/blank.data cfg/yolov3_416_batch_rotaug.cfg cfg/yolov3_416_final.weights ~/data/video1.mp4 -i 0 -out_filename video1.avi -dont_show`
